docs/index.rst

```diff
@@ -37,6 +37,7 @@ assistance.
     :hidden:
 
     installation
+    development
```

Comment:
I see, that you have explained the development procedure in great elaboration - that's really a great time taking effort.:+1:  But explain to us why is it necessary here? 

Keep in mind that documentation is "subjective" explanation so what past developers wrote will obviously vary from how you understand the concepts and write it. So rather than rephrasing, focus on what value you are adding - what specific things you are trying to solve here by re-organising content.